api/cuckoo_service.py

```diff
@@ -49,29 +53,27 @@ def upload_to_cuckoo(tracker_id, app):
     except Timeout:
         return False, "Connection timed out."
 
-def start_cuckoo_monitor(tracker_id, app):
-    log_manager = LogManager(tracker_id, app)
-    
-    try:
-        additional_data = {"error_message": "monitor start"}
-        log_manager.update_log_stage("monitor", additional_data)
-    except Exception as e:
-        print(f"Error during monitor log update: {e}")
+def start_cuckoo_monitor(tracker_id):
+    log_manager = LogManager(tracker_id)
+    print("hello")
+    additional_data = {"error_message": "monitor start"}
+    log_manager.update_log_stage("Cuckoo monitor started", additional_data)
 
-    # app_context = app.app_context()
-    # threading.Thread(target=check_cuckoo_status, args=(tracker_id, app, app_context), daemon=True).start()
+    threading.Thread(target=check_cuckoo_status, args=(tracker_id,), daemon=True).start()
 
-def check_cuckoo_status(tracker_id, app, app_context):
-    log_manager = LogManager(tracker_id, app)
-    with app_context:
-        LOG_FOLDER = app.config['LOG_FOLDER']
+def check_cuckoo_status(tracker_id):
+    print("mimo")

```

Comment:
Please remember to delete the `print` statement if you already finished the debug.